ui/app/templates/components/topo-viz/datacenter.hbs

```diff
@@ -5,11 +5,15 @@
 
 <div data-test-topo-viz-datacenter class="boxed-section topo-viz-datacenter">
   <div data-test-topo-viz-datacenter-label class="boxed-section-head is-hollow">
-    <strong>{{@datacenter.name}}</strong>
-    <span class="bumper-left">{{this.scheduledAllocations.length}} Allocs</span>
-    <span class="bumper-left">{{@datacenter.nodes.length}} Nodes</span>
-    <span class="bumper-left is-faded">{{format-bytes this.aggregatedAllocationResources.memory start="MiB"}}/{{format-bytes this.aggregatedNodeResources.memory start="MiB"}},
-      {{format-hertz this.aggregatedAllocationResources.cpu}}/{{format-hertz this.aggregatedNodeResources.cpu}}</span>
+    <span class="tooltip" aria-label="Datacenter"><strong>{{@datacenter.name}}</strong></span>
+    <span class="bumper-left tooltip" aria-label="Number of Allocations">{{this.scheduledAllocations.length}} Allocs</span>
```

Comment:
New tooltip can be used as a wrapper like this:

```
<Hds::TooltipButton @text="Datacenter" aria-label="Datacenter Name">
{{@datacenter.name}}
</Hds::TooltipButton>
```

and should be available on the latest `main`

Reply:
Awesome, that seems like a quick update, thanks!